microovn/go.mod

```diff
@@ -23,7 +23,7 @@ require (
 	github.com/go-macaroon-bakery/macaroonpb v1.0.0 // indirect
 	github.com/golang/protobuf v1.5.3 // indirect
 	github.com/google/renameio v1.0.1 // indirect
-	github.com/google/uuid v1.3.0 // indirect
+	github.com/google/uuid v1.3.1 // indirect
```

Comment:
The uuid change from v1.3.0 to v1.3.1 contains a single performance improvement fix and LGTM.